testing/integration/upgrade_fleet_test.go

```diff
@@ -98,7 +112,22 @@ func TestFleetAirGappedUpgrade(t *testing.T) {
 		Local: false, // Needed as the test requires Agent installation
 		Sudo:  true,  // Needed as the test uses iptables and installs the Agent
 	})
+	testFleetAirGappedUpgrade(t, stack, true)
```

Comment:
Can you put Unprivileged in the test name? Makes it easier to determine the reason when reading the [test report](https://buildkiteartifacts.com/e0f3970e-3a75-4621-919f-e6c773e2bb12/0187faac-2a90-4642-bc78-c33cefa167ba/018d324e-e6e7-4e9e-8dc8-4a52466581a6/018d326b-7779-4f69-8c84-d0a8a3f18d12/build/TEST-report.html?response-content-type=text%2Fhtml&X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=ASIAQPCP3C7LW7ZNQRST%2F20240122%2Fus-east-1%2Fs3%2Faws4_request&X-Amz-Date=20240122T210445Z&X-Amz-Expires=600&X-Amz-Security-Token=IQoJb3JpZ2luX2VjEJL%2F%2F%2F%2F%2F%2F%2F%2F%2F%2FwEaCXVzLWVhc3QtMSJGMEQCICYB3dvkYlM2U5cXeSxLexL%2B9W1e%2BC4nLIk87KGylgfqAiA3R01AGA2U3YeNCDwS3EicuAqgdiYG2BV8PkTp3vnF0irxAwhLEAAaDDAzMjM3OTcwNTMwMyIMcgBnsv0icBpNXN%2FBKs4DaCOPOdXPdUJ9VoPtiFvLdo%2BNkDfQLjE2eB4MY5laaL1lqH8WloggIURZu7vscA4%2F9QUjrQJm74l%2Fxi0ZeFnCvhkRgKVh2ftIz2QAEb6IKtOLyqKoxccptUaPGOiB4zr%2Bu1Ch9zmkzKNEB%2BCmn4ThbrRG39ctIkngCATLGiF1R2JjFfs9UrR9sWs0mH7mhxCmOmJ%2FwGM7d27eQegKdATJ%2FiYPxXrZpktIdnjZk1PyhKLZnkgtuQ0bYkjP3GqD48EXzcSlKQlX9fP9ue%2BLGPKazLIKFWoVh110elywwbwXjWnkKDEwFiu5NP9SFY8FNdUvbBve5l18jg9G7drw617k3S7r7Qwal2bdlID1GCnviBLvLnqKDMieE0Ewmj4FtbWs6QSF%2FekyptyfFVBqwLaSSOzHbPhaTZlgxHVeW43TNo5tjRIIna4W8mb2UrCAZHZ5QkKSVvR%2FAV%2BdI0hNCfbEsK7vjVgSMYrV4ojRHqX7oPaoSS67UnpRoanyeT3MsxpO%2FeSaq2vu%2BGo3M6wXnjKefKuF5axz%2BdyoVb3B%2Fl23f07h2vxVVoxVg%2BO90Mab%2F8s4ItIuETcuCg1RXgY49P5NJaNCX9wSu21FeTqw2AFMMN%2FXuq0GOqYB8hH21bhnDPmatoB5TvdRRpCwxLQVfHsdrWA8aHZ5Ifr9HBsoj%2FKWIfrMaNpNpUOIJNz608b3IMw4u%2F7IjPXnhmrOGtlcB6cBdwpZlAay8K0hWVcJhQh5KvASJvCYhcH2te64YEPfhWunM0qBSkPefibCaVJh6RWFKFzGmoXh2l2o%2FSfaeciOW2Jr4ouWv9TtjeANSAfZzTRKWaKdhTtzSh8vMH6d%2Fw%3D%3D&X-Amz-SignedHeaders=host&X-Amz-Signature=6bde24b1ba55b588a3f2c3dcdffbd1dc9bbdd21ae8c43a7a70c74392cfbf5a6c).

Same goes for the other tests that now have a Privileged variant, like `TestFleetManagedUpgrade(` above.